server/db/connection.js

```diff
@@ -0,0 +1,11 @@
+import mongoose from 'mongoose'
+
+export default function(app){
+    const url = "mongodb://jornie:jornie@ds047722.mlab.com:47722/soundit"
```

Comment:
add this to the .env file and share the url privately, because this exposes your database url publicly.